.github/workflows/wheels.yml

```diff
@@ -0,0 +1,43 @@
+name: Build-wheels
+on:
+  release:
+  workflow_dispatch:
+  
+jobs:
+  build_wheels:
+    name: Build wheels for cp${{ matrix.python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python: [37, 38, 39]
+        include:
+          - os: ubuntu-latest
+            cibw_before_all: yum install -y openblas-devel
```

Comment:
Out of curiosity, why are we using `yum` instead of `apt`?

Reply:
yeah this is a bit confusing - ubuntu-latest is the Github Actions runner, but cibuildwheel runs in a [manylinux](https://github.com/pypa/manylinux) container inside of that, which is based on CentOS.

Reply:
Oh, I see!